src/components/headMetaTags.js

```diff
@@ -9,16 +9,51 @@ class HeadMetaTags extends Component {
 
     let title = `${this.props.title} | Alliance of Genome Resources`;
     let data = this.props.data;
-    let schema = undefined;
+    let schemas = [];
+
+
+    // general view
+    schemas.push({
+      type: 'application/ld+json',
+      innerHTML: JSON.stringify({
+        '@context': 'http://schema.org',
+        '@type': 'Organization',
+        'url': 'https://www.alliancegenome.org',
+        'logo': 'https://pbs.twimg.com/profile_images/751097942778351616/OW7j5m0Z_bigger.jpg',
```

Comment:
Would probably be better to use an image that we host instead of our weirdly cropped twitter profile picture. Even something served from our wordpress server should be fairly easy to use instead.